examples/gno.land/r/demo/keystore/keystore_test.gno

```diff
@@ -11,10 +11,6 @@ import (
 )
 
 func TestRender(t *testing.T) {
-	// https://github.com/gnolang/gno/pull/3375 changed const -> var :
```

Comment:
This issue arose because of an invalid expression; we should use var instead of const.